R/MsBackendTimsTof-functions.R

```diff
@@ -135,4 +143,37 @@ MsBackendTimsTof <- function() {
   else unname(split.data.frame(as.matrix(tmp[, sd]), f))
 }
 
+#' Extract columns from @frames given the @indices in `x`. The function takes
+#' care of eventually duplicating values.
+#'
+#' @param x `MsBackendTimsTOF`
+#'
+#' @param columns `character` with the column names.
+#'
+#' @author Andrea Vicini, Johannes Rainer
+#'
+#' @noRd
+.get_frame_columns <- function(x, columns) {
```

Comment:
this is to avoid code duplication for all variables/columns that we want to extract from the `@frames`